docs/install/serving/install-serving-with-yaml.md

```diff
@@ -31,27 +31,28 @@ Follow the procedure for the networking layer of your choice:
 <!-- TODO: Link to document/diagram describing what is a networking layer.  -->
 <!-- This indentation is important for things to render properly. -->
 
-=== "Kourier (Choose this if you are not sure)"
```

Comment:
I think it's still useful for users to know that we recommend that they use Kourier if they don't know which layer to install. Could you replace this information?